integration/kafka/ReadMe.md

```diff
@@ -0,0 +1,111 @@
+##Kafka Alluxio Connect
+Kafka Alluxio Connector implements exporting kafka's specific topic data to alluxio filesystem.User can define data format stored in alluxio file,such as JsonFormat.
+##Perquisites
+
+- confluent-2.0.1
+- alluxio-1.0.1
+
+##Configuration
+###Alluxio Connector Configuration 
+####Example
+
+Here is the content of `etc/kafka-connect-alluxio/alluxio-connect.properties`
+    
+
+	name=alluxio-sink
+	connector.class=alluxio.kafka.connect.AlluxioSinkConnector
+    tasks.max=1
+    topics=test_alluxio
+    alluxio.url=alluxio://localhost:19998
+    topics.dir=topics
+    rotation.num=3
+    rotation.time.interval=-1
+    alluxio.format=alluxio.kafka.connect.format.JsonFormat
+
+
+User can configure connector class in this property file,the `topics` specify the topics we want export data from kafka. `alluxio.url` is the alluxio address where we export data to. `topics.dir ` is the top level directory in alluxio filesystem where the kafka data is stored.`rotation.num` specify the kafka record number writen in alluxio file before file closes.`rotation.time.interval` is similar to `rotation.num`, alluxio file will be closed if the time interval between up-to-date write and last rotatime time is beyond `rotation.time.interval`. ` alluxio.format` specify the serialized data fomat stored in alluxio file.
+###Connector Worker Configuration
+####Example
+Here is the sample configuration for a standalone Kafka Connect worker that uses Avro serialization and integrates the the Schema Registry. This sample configuration assumes a local installation of Confluent Platform with all services running on their default ports.
+	
+	# Bootstrap Kafka servers. If multiple servers are specified, they should be comma-separated.	
+	bootstrap.servers=localhost:9092
+	key.converter=io.confluent.connect.avro.AvroConverter
+	key.converter.schema.registry.url=http://localhost:8081
+	value.converter=io.confluent.connect.avro.AvroConverter
+	value.converter.schema.registry.url=http://localhost:8081
+	# Local storage file for offset data
+	offset.storage.file.filename=/tmp/connect.offsets
+
+Confluent Connector and Worker configuration details can be seen [Kafka connector configuration](http://docs.confluent.io/2.0.1/connect/userguide.html#configuring-connectors)
+##Setup
+- Install and run Alluxio
```

Comment:
Can you add a link here (e.g. http://alluxio.org/documentation/master/en/Getting-Started.html)